src/main/scala/com/sagacify/sonar/scala/ScalaSensor.scala

```diff
@@ -9,14 +9,14 @@ import scalariform.ScalaVersions
 
 class ScalaSensor(scala: Scala) extends Sensor {
 
-
   private val ScalaVersionPropertyKey = "sonar.scala.version"
 
   override def execute(context: SensorContext): Unit = {
     val charset = context.fileSystem().encoding.toString
     val versionProperty = context.config().get(ScalaVersionPropertyKey)
     val version =
-      if (context.config().get(ScalaVersionPropertyKey).isPresent) versionProperty.get() else ScalaVersions.Scala_2_11.toString()
+      if (context.config().get(ScalaVersionPropertyKey).isPresent) versionProperty.get()
+      else ScalaVersions.Scala_2_11.toString()
```

Comment:
Oh, `versionProperty ` is a Java's `Optional`... Maybe including https://github.com/scala/scala-java8-compat as a dependency would make our life a bit easier in the future when dealing with Scala <-> Java conversions?
With this library the above would become:
```scala
versionProperty.asScala.getOrElse(ScalaVersions.Scala_2_11.toString)
```

Reply:
Oh I really don't know, this was just a scalafmt change, I didn't touch it.

Reply:
I raised a new issue to address this - https://github.com/mwz/sonar-scala/issues/24.